.github/workflows/ci.yml

```diff
@@ -13,16 +13,18 @@ env:
   RUST_BACKTRACE: "full"
   # Increase thread stack size to 8 megabytes.
   RUST_MIN_STACK: 8000000
+  # Fail on documentation warnings.
+  RUSTDOCFLAGS: '-D warnings'
```

Comment:
can this be passed in like we do with clippy or can you not do that with `doc`?

Reply:
You cannot - we could do `RUSTDOCFLAGS='-D warnings' cargo doc` but that causes a force-rebuild, so it's set for the whole CI environment.

Reply:
ah alright yeah this is better then